cpp/test/comm/CMakeLists.txt

```diff
@@ -28,27 +28,45 @@ function (add_unit_test)
     cxx_target_compile_options (AppleClang ${name} PRIVATE -DBOOST_ASIO_HAS_STD_CHRONO)
 endfunction()
 
-
 # Build common code into its own library.
```

Comment:
This one is more than just shared generated code. It also has shared `main()` and some other things.